### ISSUE TITLE
Text Description update polling

### ISSUE THREAD

**Issue by JakeWags** [visdesignlab/upset2#493]
Currently, when the sidebar is closed, there are NO alt-txt polls, which is fine for performance; however, when you open the sidebar, the text does NOT match the plot state. There should be a poll when the sidebar is opened. 

Additionally, when the user "edits" the text, but makes no changes, the state saves as though they did make changes, and so the text no longer updates with the plot.
